books/RayTracingTheNextWeek.html

```diff
@@ -94,10 +93,35 @@
 </div>
 
 
+Managing Time
+--------------
+Before continuing, let's think about time, and how we might manage it across one or more renders.
+There are two aspects of shutter timing to think about: the time from one shutter opening to the
+next shutter opening, and how long the shutter stays open for each frame. Standard movie film used
+to be shot at 24 frames per second. Modern digital films can be 30 or even 60 frames per second.
+Each frame can have its own shutter speed, which need not be -- and typically isn't -- the maximum
+duration of the entire shutter period. You could have the shutter open for 1/1000th of a second per
+frame, or 1/60th of a second. It may surprise you to learn that for most film motion pictures, the
+screen is actually dark more than it is lit up with the movie frames!
```

Comment:
With tech like Black frame insertion and similar, the amount of time that a frame is lit up has more to do with brightness levels than shutter speeds.

Shutter speeds are orthogonal to frame on time.
If you think about it, unless the codec has metadata, the TV has no way of knowing what the shutter speed of any given frame was.

And if the shutter is closed more than it is open, it's due to a quirk of the way films are shot. (I will spare you the specifics)

Consider rewriting to
"It may surprise you to learn that for most motion pictures, the shutter is actually closed as much -- if not more -- than it is open"

Reply:
You'll note that I made a mistake above, which confuses things even more. Corrected:

“... its own shutter speed, which need not be -- and typically isn't -- the maximum duration of the entire ~shutter period~ frame.”

My intent here was to really drive home the distinction between frame rate and shutter speed, because both would need to be managed if readers when that route and wanted to simulate physical cameras, and just to share/break some assumptions. But this is kind of rat-holing, and probably unnecessary.

> And if the shutter is closed more than it is open, it's due to a quirk of the way films are shot. (I will spare you the specifics)
I presume you're thinking of the circular shutter plate. Yeah, and it's really cool seeing the almost subliminal perception as you mess with these things. But I'm thinking now I should cut this short.

Reply:
| Circular shutter plate

Not exactly. The circular shutter plate is the reason you typically want shutter time to be half of the frame time.
For good-looking motion blur.

No. The reason that some films are closed more than they are open is because hey shoot at 24 frames per second, but have a shutter speed of _1/50_ of a second.

Cinematography was a ***mistake***